pages.ja/common/aws.md

```diff
@@ -0,0 +1,37 @@
+# aws
+
+> アマゾンウェブサービスの公式 CLI ツールです。
+> `aws S3`などいくつかのサブコマンドは、独自の使用方法に関するドキュメントが用意されています。
```

Comment:
```suggestion
> `aws S3` のようないくつかのサブコマンドには、使用方法についての独自のドキュメントがあります。
```
This is what we use for Japanese pages. [Ref](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md)

Reply:
@kbdharun 
Thanks for the review!
It looks good to me, so I'll adopt it.